eo-phi-normalizer/src/Language/EO/Phi/Dataize.hs

```diff
@@ -184,27 +184,33 @@ evaluateBinaryDataizationFunChain ::
   EvaluationState ->
   DataizeChain (Object, EvaluationState)
 evaluateBinaryDataizationFunChain name resultToBytes bytesToParam wrapBytes arg1 arg2 func obj _state = do
-  let lhsArg = arg1 obj
-  let rhsArg = arg2 obj
-  lhs <- incLogLevel $ do
-    logStep "Evaluating LHS" (Left lhsArg)
-    dataizeRecursivelyChain True lhsArg
-  rhs <- incLogLevel $ do
-    logStep "Evaluating RHS" (Left rhsArg)
-    dataizeRecursivelyChain True rhsArg
-  result <- case (lhs, rhs) of
-    (Right l, Right r) -> do
-      let bytes = resultToBytes (bytesToParam l `func` bytesToParam r)
-          resultObj = wrapBytes bytes
-      logStep "Evaluated function" (Left resultObj)
-      return resultObj
-    (Left _l, Left _r) ->
-      fail (name <> ": Couldn't find bytes in both LHS and RHS")
-    (Left l, _) -> do
-      fail (name <> ": Couldn't find bytes in LHS: " <> printTree (hideRho l))
-    (_, Left r) -> do
-      fail (name <> ": Couldn't find bytes in RHS: " <> printTree (hideRho r))
-  return (result, ())
+  ctx <- getContext
+  if HashSet.member (DisabledAtomName name) ctx.disabledAtomNames
```

Comment:
This is not the correct place for this check. Instead, use the check in the `dataizeStepChain` function, before `evaluateBuiltinFunChain` is called.